config/environments/production.rb

```diff
@@ -17,51 +19,49 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
-  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
-  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
-  # `config/secrets.yml.key`.
-  config.read_encrypted_secrets = true
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
```

Comment:
Should I carry this over?

Reply:
shouldn't be needed I dont think. does the JS render?

Reply:
Yes the JS "Does" render. I can see the progress bar js and the ajax script work for when an importer is currently running in the bg job.

Reply:
@aprilrieger Let's do some more testing around this

Reply:
This is a production file, so me testing locally isn't going to be sufficient. I am sticking it back in for now.